src/Command/DrupalUpdate/DrupalPackageManager.php

```diff
@@ -0,0 +1,314 @@
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class DrupalPackageManager {
+
+  /**
+   * @var OutputInterface
+   */
+  private OutputInterface $output;
+
+  /**
+   * @var array
+   */
+  private array $packageData=[];
+
+  /**
+   * @var bool
+   */
+  private bool $isCoreUpdated;
+
+  /**
+   * @var SymfonyStyle
+   */
+  private SymfonyStyle $io;
+
+  /**
+   * @var array
+   */
+  public array $infoPackageFilesPath=[];
+
+  /**
+   * @var FileSystemUtility
+   */
+  private FileSystemUtility $fileSystemUtility;
+
+  /**
+   * @var DrupalOrgClient
+   */
+  private DrupalOrgClient $drupalOrgClient;
+
+  /**
+   * @return DrupalOrgClient
+   */
+  public function getDrupalOrgClient(): DrupalOrgClient {
+    return $this->drupalOrgClient;
+  }
+
+  /**
+   * @param DrupalOrgClient $drupalOrgClient
+   */
+  public function setDrupalOrgClient(DrupalOrgClient $drupalOrgClient): void {
+    $this->drupalOrgClient = $drupalOrgClient;
+  }
+
+  /**
+   * @param bool $isCoreUpdated
+   */
+  public function setIsCoreUpdated(bool $isCoreUpdated): void {
+    $this->isCoreUpdated = $isCoreUpdated;
+  }
+
+  /**
+   * @return array
+   */
+  public function getPackageData(): array {
+    return $this->packageData;
+  }
+
+  /**
+   * @param OutputInterface $output
+   */
+  public function setOutput(OutputInterface $output): void {
+    $this->output = $output;
+  }
+
+  /**
+   * @param FileSystemUtility $fileSystemUtility
+   */
+  public function setFileSystemUtility(FileSystemUtility $fileSystemUtility): void {
+    $this->fileSystemUtility = $fileSystemUtility;
+  }
+
+  public function __construct(InputInterface $input, OutputInterface $output) {
+    $this->setIsCoreUpdated(FALSE);
+    $this->setDrupalOrgClient(new DrupalOrgClient($input, $output));
+    $this->setOutput($output);
+    $this->io = new SymfonyStyle($input, $output);
+    $this->setFileSystemUtility(new FileSystemUtility($input, $output));
+  }
+
+  /**
+   * @param string $drupal_project_cwd_path
+   * @return mixed
+   */
+  protected function determineCorePackageVersion(string $drupal_project_cwd_path): mixed {
+    $drupal_boostrap_inc_path = $drupal_project_cwd_path . '/docroot/includes/bootstrap.inc';
+    if (file_exists($drupal_boostrap_inc_path)) {
+      $boostrap_inc_file_contents = file_get_contents($drupal_boostrap_inc_path);
+      preg_match("/define\(\s*'([^']*)'\s*,\s*'([^']*)'\s*\)/i", $boostrap_inc_file_contents, $constraint_matches);
+      if ((count($constraint_matches) > 2) && ($constraint_matches[1] == 'VERSION')) {
+        return $constraint_matches[2];
+      }
+    }
+    return "";
+  }
+
+  /**
+   * @param array $info_packages_file_path
+   * @param string $drupal_project_cwd_path
+   * @throws AcquiaCliException
+   */
+  public function getPackageDetailInfo(array $info_packages_file_path, string $drupal_project_cwd_path): void {
```

Comment:
This function does not return anything. The name of the function should start with "set" instead of get.